calicoctl/commands/ipam.go

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/tigera/libcalico-go/lib/client"
+	caliconet "github.com/tigera/libcalico-go/lib/net"
+
+	docopt "github.com/docopt/docopt-go"
+)
+
+// IPAM show/release is for managing Calico IPAM by showing the
```

Comment:
This comment is wrong - it shouldn't talk about "show/release" (or any other specific IPAM subcommands), since this is the general comment for the entire IPAM func.